MapboxDirections/MBDirections.swift

```diff
@@ -172,6 +175,7 @@ open class Directions: NSObject {
                 for route in routes {
                     route.accessToken = self.accessToken
                     route.apiEndpoint = self.apiEndpoint
+                    route.routeIdentifier = self.routeIdentifier
```

Comment:
Get the route identifier from `json["uuid"]` instead of `Directions.routeIdentifier`.